src/hypertable_cache.h

```diff
@@ -12,10 +12,11 @@
 #include "cache.h"
 #include "hypertable.h"
 
-extern TSDLLEXPORT Hypertable *ts_hypertable_cache_get_entry(Cache *cache, const Oid relid,
+extern TSDLLEXPORT Hypertable *ts_hypertable_cache_get_entry(Cache *const cache, const Oid relid,
```

Comment:
This is to silence warnings by `MSVC`, which otherwise complains about declaration and definition being different. I do think there's a point to keeping these in sync too, even if what you point out is strictly true.

Reply:
Sounds like an error in MSVC, but yeah, we have to handle these as well so in that case you have to keep it.